.github/ISSUE_TEMPLATE/offboarding-request.md

```diff
@@ -18,21 +18,24 @@ Please provide the following information about the individual being offboarded:
 - name:
   > Janet Doe
 - team name:
-  > 
+  >
 - role on team:
-  > 
+  >
 - expected last day: _(MM/DD/YYYY)_
-  > 
+  >
 - company they work for:
-  > 
+  >
 - email: _(Multiple emails listed in order of relevance to VA business, VA, Company, Personal, etc....)_
-  > 
+  >
 - GitHub username: _(Please include @ symbol)_
-  > 
+  >
 - additional access they may have: _(First line formatted with access titles only. Additional notes may be added on second line after the list.)_
   > ex: SOCKS, Pagerduty, Google analytics, etc.
- 
- 
+
+<br /><br />
+:x: All done. Don't fill out anything below here. :x:
```

Comment:
🙌 🎉 Just to be more explicit, how about this?

```suggestion
:x: All done. Don't fill out or modify anything below here. :x:
```

Also 🤔 I don't think you need the `br` tags. Leaving blank lines _should_ create a new paragraph, if that was your intention here.

Reply:
The `br` is there because a the normal spacing between paragraphs didn't seem to make the message stand out enough. I like your "🙌 🎉" though, so lets go with that and no `br`s and see if it's sufficient :)